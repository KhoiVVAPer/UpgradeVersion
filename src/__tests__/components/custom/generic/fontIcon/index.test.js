import React from 'react';
import renderer from 'react-test-renderer';
import { shallow } from 'enzyme';
import FontIcon from '../../../../../components/custom/generic/fontIcon/_fontIcon';

const setUp = (props) => {
  const component = shallow(<FontIcon {...props} />);
  return component;
};

describe('components > custom > elements > FontIcon', () => {
  let component;
  const props = {
    icon: 'star',
    type: 'Entypo',
  };
  beforeEach(() => {
    component = setUp(props);
    return component;
  });

  it('Check snapshot', () => {
    const tree = renderer.create(component).toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('Renders FontWrap component', () => {
    const input = component.find('FontWrap').length;
    const output = 1;
    expect(input).toEqual(output);
  });
});
