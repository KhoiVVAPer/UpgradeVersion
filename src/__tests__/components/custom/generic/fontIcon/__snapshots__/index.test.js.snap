// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components > custom > elements > FontIcon Check snapshot 1`] = `
<react-native-mock>
  <react-native-mock>
    ï‰—
  </react-native-mock>
</react-native-mock>
`;
